src/main/java/uk/ac/ebi/eva/evaseqcol/controller/seqcol/SeqColComparisonController.java

```diff
@@ -0,0 +1,62 @@
+package uk.ac.ebi.eva.evaseqcol.controller.seqcol;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import uk.ac.ebi.eva.evaseqcol.entities.SeqColComparisonResultEntity;
+import uk.ac.ebi.eva.evaseqcol.entities.SeqColEntity;
+import uk.ac.ebi.eva.evaseqcol.entities.SeqColLevelTwoEntity;
+import uk.ac.ebi.eva.evaseqcol.exception.SeqColNotFoundException;
+import uk.ac.ebi.eva.evaseqcol.service.SeqColService;
+
+import java.io.IOException;
+
+@RestController
+@RequestMapping("/comparison")
+public class SeqColComparisonController {
+
+    private SeqColService seqColService;
+
+    @Autowired
+    public SeqColComparisonController(SeqColService seqColService) {
+        this.seqColService = seqColService;
+    }
+
+    @GetMapping("/{digest1}/{digest2}")
```

Comment:
**Optional**: Add links to the corresponding parts of the sequence-collection specification in a comment